### ISSUE TITLE
Test seems to fail depending on host filesystem

### ISSUE THREAD

**Issue by DeeUnderscore** [dundee/gdu#58]
[`TestFlags()`](https://github.com/dundee/gdu/blob/0a48ba3d55c83726c091b0764ae2c150da8b087e/pkg/analyze/dir_test.go#L82) seems to fail depending on what filesystem the repository is checked out to. When doing `make test` from the repo on an ext4 volume, all tests pass. On the other hand, on a btrfs volume, I get a failure:

```
    dir_test.go:82: 
                Error Trace:    dir_test.go:82
                Error:          Not equal: 
                                expected: 0
                                actual  : 4096
                Test:           TestFlags
--- FAIL: TestFlags (0.00s)
```

(this is in both cases with go 1.16.5 on NixOS 21.05, Linux 5.10.37)

This particular test also fails on Hydra, the NixOS build server, with yet different output: https://hydra.nixos.org/build/145014299/nixlog/1

**Comment by dundee** [comment]
Should be fixed in 5.1.1.

**Comment by DeeUnderscore** [comment]
Thanks for tagging the release!
